integreat_cms/cms/forms/language_tree/language_tree_node_form.py

```diff
@@ -119,3 +121,19 @@ def clean(self):
             "LanguageTreeNodeForm validated [2] with cleaned data %r", cleaned_data
         )
         return cleaned_data
+
+    def save(self, commit=True):
+        """
+        This method extends the default ``save()``-method of the base :class:`~django.forms.ModelForm` to flush
+        the cache after commiting.
+
+        :param commit: Whether or not the changes should be written to the database
+        :type commit: bool
+
+        :return: The saved page translation object
+        :rtype: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
+        """
+        # Save CustomModelForm and flush Cache
+        result = super().save(commit=commit)
+        invalidate_model(LanguageTreeNode)
```

Comment:
```suggestion
        invalidate_model(Page)
        invalidate_model(Event)
        invalidate_model(POI)
```
This already happens implicitly here:
https://github.com/digitalfabrik/integreat-cms/blob/8c06962b2549f3c6839bc38b4b8f25d95a6ce7e6/integreat_cms/cms/models/abstract_tree_node.py#L296-L300
Each node is "moved" after saving, no matter if the position changed or not.

However, I guess it would be good to invalidate all content models when the language tree is changed. I think cacheops would do this automatically if the translation models had a foreign key to the language tree node model, but since they're just related to the language model, cacheops doesn't think they have anything to do with the language tree. Maybe it's also required to invalidate the translation models, but I presume they are implicitly invalidated because they have a foreign key relation to the content models.